ipatests/test_integration/test_pwpolicy.py

```diff
@@ -347,7 +344,6 @@ def test_graceperiod_not_replicated(self):
         self.reset_password(self.master)
 
         result = tasks.ldapsearch_dm(
-            self.replicas[0], dn, ['passwordgraceusertime',],
+            self.master, dn, ['passwordgraceusertime',],
```

Comment:
lines 337-343 and 344-349 are the same (reset pwd on master, check value on master), did you mean to reset and check on the replica instead?

Reply:
You're right, it's extraneous. It will only update on the master so there is no need to check the replica, which we just did by ensuring that incrementing it didn't cause replication.

I've removed them.